src/gui/application.cpp

```diff
@@ -764,8 +765,14 @@ void Application::handleEditLocally(const QUrl &url) const
     // for a sample URL "nc://open/admin@nextcloud.lan:8080/Photos/lovely.jpg", QUrl::path would return "admin@nextcloud.lan:8080/Photos/lovely.jpg"
     const auto accountDisplayName = pathSplit.takeFirst();
     const auto fileRemotePath = pathSplit.join('/');
+    QRegularExpression tokenRegexp(QStringLiteral("^token=(.*)$"));
 
-    FolderMan::instance()->editFileLocally(accountDisplayName, fileRemotePath);
+    const auto tokenMatch = tokenRegexp.match(url.query());
+    if (tokenMatch.hasMatch()) {
+        FolderMan::instance()->editFileLocally(accountDisplayName, fileRemotePath, tokenMatch.captured(1));
```

Comment:
@mgallien It would make sense to process an empty token inside `void FolderMan::editFileLocally` and display an error message box to a user along with this log message